keypoints/include/pcl/keypoints/impl/iss_3d.hpp

```diff
@@ -252,8 +249,7 @@ pcl::ISSKeypoint3D<PointInT, PointOutT, NormalT>::initCompute ()
 
     delete[] third_eigen_value_;
 
-  third_eigen_value_ = new double[input_->size ()];
-  memset(third_eigen_value_, 0, sizeof(double) * input_->size ());
+  third_eigen_value_ = new double[input_->size ()]{};
```

Comment:
Not sure about the performance, but couldn't you use here `std::vector`? Same below